Tests/FormGeneratorTest.php

```diff
@@ -58,6 +58,26 @@ public function testNamedForm()
         $this->assertSame('my_form', $form->getName());
     }
     
+
+    public function testNamedFormWithOptionsMethodGet(){
```

Comment:
I think that one test per method will be enough, they're checking the same thing anyway. Also please keep the coding style, `{` should go to new line, mind spaces around `(` and `,`